src/main/java/io/ebeaninternal/server/deploy/parse/AnnotationBase.java

```diff
@@ -321,7 +319,9 @@ private static final Method getNullMethod() {
     for (T ann : anns) {
       try {
         if (getPlatformsMethod == null) {
-          getPlatformsMethod = ann.getClass().getMethod("platforms");
+          // Google error prone says: Calling getClass() on an annotation may return a proxy class, use annotationType() instead.
+          // http://errorprone.info/bugpattern/GetClassOnAnnotation
+          getPlatformsMethod = ann.annotationType().getMethod("platforms");
```

Comment:
Google's "Error Prone" compiler pointed out only 1 error prone thing about ebean's code and this was it.

Details here: http://errorprone.info/bugpattern/GetClassOnAnnotation